tools/challenge-editor/client/src/components/editor/editor.tsx

```diff
@@ -87,6 +88,15 @@ const Editor = () => {
       <p>
         <Link to={`/${superblock || ''}/${block || ''}`}>Return to Block</Link>
       </p>
+      <p>
+        <Link
+          to={`localhost:8000/learn/${superBlockNameMap[superblock || '']}/${block || ''}/${items.name.replace(/[\s]+/g, '-').toLowerCase() || ''}`}
```

Comment:
I could be wrong, but I believe `HOME_LOCATION` is available in Gitpod. If so, we should use that since localhost won't work.

Reply:
Hey Oliver, I found out that the environment variable file for the tools is different from the environment variable for the main application. I had to define a new one with the prefix `CHALLENGE_EDITOR_`. I decided to use the name `CHALLENGE_EDITOR_FCC_LOCATION` to mean where the FreeCodeCamp application lives and am using a fallback variable of `localhost:8000`. Is this acceptable?

Reply:
Hiya. Yeah, the challenge editor has its own .env file and doesn't draw on the main one (where HOME_LOCATION is usually defined). However, Gitpod is a little different: it puts a few env vars into the environment making them available to all apps, including the challenge editor:

https://github.com/freeCodeCamp/freeCodeCamp/blob/f82aff217bfea9ae0c1e5a8039bb7ee6722713b0/.gitpod.yml#L31-L38

I don't know the best way to handle this in Vite, but adding another var like `CHALLENGE_EDITOR_LEARN_CLIENT_LOCATION=$(gp url 8000)` to `.gitpod.yml` should do the trick.

Reply:
It's in the Yml file. I don't use gitpod enough to tell if it's working.